libs/application/templates/children-residence-change/src/fields/Overview/index.tsx

```diff
@@ -47,7 +47,7 @@ const Overview = ({ application }: FieldBaseProps) => {
 
   return (
     <>
-      <Box marginTop={3}>
+      <Box marginTop={0}>
```

Comment:
I changed this because we now have a `buildMultiField` from the application system 🤦  and that has a mandatory description 🤦  which I set to an empty string but it of course takes up vertical space. 🤦 
So even though marginTop here is set to zero it is still a little bit off. Thinking of having this as an TODO to look at later, any thoughts?

Reply:
The only thing I can think of right now is not using this `buildSubmitField` and create our own. But I guess this will do for now..

Reply:
Yes agree, but it is big refactor since we need to replace the footer as well then and have our own custom custom component. But I think it is definitely a todo and the sooner the better 🙏 
IMO then this approach is a patch and then we will do it the right way when we have the time.